docs/infrastructure/index.asciidoc

```diff
@@ -15,15 +13,58 @@ image::infrastructure/images/infra-sysmon.jpg[Infrastructure Overview in Kibana]
 
 
 [float]
-== Add data sources
-Kibana provides step-by-step instructions to help you add your data sources.
-The {infra-guide}[Infrastructure Monitoring Guide] is a good source for more detailed
-instructions and information.
+== Add data
+
+Kibana provides step-by-step instructions to help you add log data. The
+{infra-guide}[Infrastructure Monitoring Guide] is a good source for more
+detailed information and instructions.
 
 [float]
 == Configure data sources 
 
-By default the Infrastructure UI uses the `metricbeat-*` index pattern to query the data. If you configured Metricbeat to export data to a different set of indices, you will need to set `xpack.infra.sources.default.metricAlias` in `config/kibana.yml` to match your index pattern. You can also configure the timestamp field by overriding `xpack.infra.sources.default.fields.timestamp`. See <<infrastructure-ui-settings-kb>> for a complete list.
+By default the Infrastructure UI uses the `metricbeat-*` index pattern to query
+the data.  If your metrics are located in a different set of indices or use a
+different timestamp field, you can adjust the source configuration via the user
+interface or the {kib} configuration file.
+
+NOTE: The Logs and Infrastructure UI share a common data source definition in
+each space. Changes in one of them can influence the data displayed in the
+other.
+
+[float]
+=== Configure data sources in the user interface
```

Comment:
```suggestion
=== Configure data sources in Kibana
```

Reply:
The intention was to point out the contrast between configuring via the UI and configuring via the config file: "Configure data sources in the user interface" vs "Configure data sources in the configuration file"

With "Configure source" vs "Configure data sources in the configuration file" the distinction is less clear.

Do you think that this parallelism would be worth preserving or should we give up on that?

Reply:
My thought process behind this suggestion:
1) We want to try to avoid saying UI or user interface when documenting a UI. It's assumed that's what we're referring to.
2) The top level heading of this section is titled _Configure data sources_, and it explains that there are two ways to ocnfigure data source: source config, or the config file. I don't think we need to repeat the text "configure data sources" in each of the child headings.

Based on ^^ - I think the best layout would be:
* Configure data sources
  * Configure source
  * Configuration file

Reply:
Ok, that makes sense... I wasn't thinking of this as just the UI docs, but more as a "feature documentation" that consists of both frontend and backend. Maybe that's too much of a technical view. :innocent: